changelog

```diff
@@ -1,3 +1,400 @@
+      * Human jobs requiring brigs or luxury accommodations will now appear in the job board but be blocked if the outfit is not present. (@bene-dictator)
+      * Improved text in Remnant: Salvage 4 to better describe mission objective and failure condition. (@ThrawnCA)
+      * Improved some writing in FW Southern Battle 1B. (@Zitchas)
+      * The Behemoth bounty target in FW Bounty 2 is now a Mammoth. (@1010todd, @Saugia)
+    * Balance:
+      * Adjusted the Sparrow and Flivver to better match their roles. (@Hecter94)
+        * Flivver:
+          * +10 cargo sapce (5 -> 15)
+          * -9 weapon capacity (25 -> 16)
+          * Remove its two Meteor Missile Pods
+        * Sparrow:
+          * +200 shields (1200 -> 1400)
+          * +5 weapon capacity (20 -> 25)
+      * Made the Falcon and Leviathan more distinct from each other. (@Anarchist2)
+        * Falcon:
+          * +16 crew (75 -> 91)
+          * +40 cargo space (90 -> 130)
+          * +20 outfit space (540 -> 560)
+          * +10 weapon capacity (240 -> 250)
+        * Leviathan:
+          * +120 mass (480 -> 600)
+          * +0.5 drag (7.6 -> 8.1)
+        * Marauder Falcon:
+          * +14 requires crew (43 -> 57)
+          * +16 bunks (80 -> 96)
+          * +20 cargo space (60 -> 80)
+          * +40 outfit space (580 -> 620)
+          * +10 weapon capacity (265 -> 275)
+        * Marauder Leviathan:
+          * +40 mass (640 -> 680)
+          * +0.5 drag (7.6 -> 8.1)
+      * Increased the sizes and damage of some human secondary weapons. (@Pointedstick)
+        * TODO: this
+      * Increased the outfit space of the Marauder Manta (390 -> 400). (@Pointedstick)
+      * Increased the outfit space and weapon capacity of the Rainmaker (230 -> 240 and 60 -> 70). (@Pointedstick)
+      * Buffed the Star Queen. (@Anarchist2)
+        * -1300000 base cost (5.5M -> 4.2M)
+        * -8 requires crew (41 -> 33)
+      * The Rano'ereks that appear as miners in human space are now better able to defend themselves. (@ravenshining)
+      * Made the battle in Terraforming Tundra less difficult. (@Anarchist2)
+      * Added a Small Biochemical Cell to the Summer Leaf. (@warp-core)
+      * Added some more Wanderer escort ships (equipped with Moonbeams) to "Wanderers: Sestor: Factory Escorts". (@warp-core)
+      * Significantly increase the energy capacity of Archons. (@Zitchas)
+      * Nerfed the flamethrower. (@Quantumshark)
+        * +100% firing heat (2 -> 4).
+        * -25% heat damage (200 -> 150)
+        * Added damage dropoff with range beyond 45 units. Damage is halved by maximum range.
+      * Increased the size of some Large Syndicate fleet variants. (@Quantumshark)
+      * Added a small amount of inertia reduction and force protection to Quarg engines. (@Zitchas)
+      * Made the Unfettered Hai attacking Wah Ki cowards and the Hai defending the system merciful, so fewer Unfettered die there. (@Hurleveur)
```

Comment:
```suggestion
      * Made the Unfettered Hai attacking Wah Ki cowards and the Hai defending the system merciful, so fewer Unfettered get disabled and die there. (@Hurleveur)
```
The main purpose of it was balance so I feel like we should mention this

Reply:
```suggestion
      * Made the Unfettered Hai attacking Wah Ki cowards and the Hai defending the system merciful, so fewer Unfettered get disabled or die there. (@Hurleveur)
```

Reply:
That works too sure